fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/observer/eventmappings/AllNodeEventsOneEvent.java

```diff
@@ -35,14 +36,12 @@
 
 import org.slf4j.Logger;
 
-import com.google.common.base.Function;
 import com.google.common.collect.Multimap;
 
 /**
- * Maps all JCR {@link Event}s concerning one JCR node to one
- * {@link FedoraEvent}. Adds the types of those JCR events together to calculate
- * the final type of the emitted FedoraEvent. TODO stop aggregating events in
- * the heap and make this a purely iterative algorithm, if possible
+ * Maps all JCR {@link Event}s concerning one JCR node to one {@link FedoraEvent}. Adds the types of those JCR events
+ * together to calculate the final type of the emitted FedoraEvent. <br/>
```

Comment:
This <br/> tag causes an error with `mvn javadoc:jar` (which is why travis fails). If you could remove it, I think this PR is good to go.